src/applications/pensions/containers/ConfirmationPage.jsx

```diff
@@ -33,77 +27,124 @@ class ConfirmationPage extends React.Component {
 
     return (
       <div>
-        <h3 className="confirmation-page-title">Claim submitted</h3>
-        <p>
-          We process claims in the order we receive them. Please print this page
-          for your records.
-        </p>
-        <p>We may contact you for more information or documents.</p>
+        <h2>Your Veteran's Pension application</h2>
+
+        <va-alert status="success">
+          <h3>Thank you for submitting your Veterans Pension application.</h3>
+          <p>
```

Comment:
This utility class will follow the suggested layout for v3 alerts.
```suggestion
          <p className="vads-u-margin-y--0">
```